src/components/candidates/CandidatesList.vue

```diff
@@ -120,16 +118,24 @@ getCandidates();
             <p class="border-dark fs-5 m-0">
               Position: {{ candidate.position }}
             </p>
-            <button class="btn btn-primary mt-4">Open candidate</button>
+            <button class="btn btn-primary mt-4">See full description</button>
           </div>
         </router-link>
       </li>
     </ul>
     <Pagination
-      v-if="!isLoaderVisible"
+      v-if="!isLoaderVisible && candidatePagesNum > 1"
       v-model:currentPage="currentPage"
       class="pb-5 pt-2"
       :page-count="candidatePagesNum"
     />
   </div>
 </template>
+<style scoped>
```

Comment:
Use `<style lang="sass">` it's shorter

Reply:
class namespace can be duplicated in candidateView and CandidatesListView, so i used scoped in here